config/views.py

```diff
@@ -19,6 +25,56 @@ def get_context_data(self, **kwargs):
         }
 
 
+class CourtsTribunalsView(TemplateViewWithContext):
+    template_name = "pages/courts_and_tribunals.html"
+    page_title = "Judgments and decisions by court or tribunal"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["context"]["courts"] = courts.get_grouped_selectable_courts()
+        context["context"]["tribunals"] = courts.get_grouped_selectable_tribunals()
+        context["feedback_survey_type"] = "courts_and_tribunals"
+        return context
+
+
+class CourtOrTribunalView(TemplateViewWithContext):
+    template_name = "pages/court_or_tribunal.html"
```

Comment:
Given the slightly confusingly named classes, doc strings to disambiguate the list from the individual might be nice.